packages/crawler/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "crawler",
+  "version": "0.0.0",
+  "private": true,
+  "license": "UNLICENSED",
+  "scope": "@porsche-design-system",
+  "author": "Dr. Ing. h.c. F. Porsche AG",
+  "scripts": {
+    "start": "ts-node src/index.ts",
+    "test:e2e:puppeteer": "jest --colors -c tests/e2e/config/jest.config.ts",
```

Comment:
This could be simplified by using the puppeteer setup from our shared package. There, by passing a `PORT` env variable the setup is going to execute `yarn start-app` and wait for that port to be ready.
See other packages for reference.

Reply:
@denyo I saw such implementation in other packages, but in this case we don't need "start-app", because we don't need any app (web server) to be running for e2e test, only cdn.. Therefore, we don't need any port to be passed and we can't wait till port is ready

Reply:
@denyo I thought about it the second time, and I will try using shared package config, just without PORT (since we don't have web server running)

Reply:
I changed the implementation - now we use shared package config. We don't pass PORT here, because we don't need the web server.

Reply:
I suppose with concurrently starting the cdn and executing the tests in parallel everything works. 
However this will probably kill interactive testing but definitely destroy passing additional parameters.  
A way around this could be to pass the `PORT` env variable and add a `start-app` command which then calls `serve-cdn`?

Reply:
Changed it